src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0,
+        maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="product-card" data-qa="card">
+      <img class="product-card__image" src="./images/imac.jpeg"
+        alt="Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A) picture">
+      <div class="product-card__name-code name-code">
+        <h2 class="name-code__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h2>
+        <p class="name-code__code">Код товара: 195434</p>
+      </div>
+      <div class="product-card__rating rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="rating__customers-coments">
+          Отзывов: 5
+        </div>
+      </div>
+      <div class="product-card__price-label price-label">
```

Comment:
BEM block should be an a block that can be re-used in another place. 2 lines with price most probably won't be reused